CorsixTH/Lua/objects/reception_desk.lua

```diff
@@ -91,11 +91,15 @@ function ReceptionDesk:onClick(ui, button)
   end
 end
 
+local queue_rerouted = false
+
 function ReceptionDesk:tick()
   local queue_front = self.queue:front()
   local reset_timer = true
   if self.receptionist and queue_front then
-    if queue_front:getCurrentAction().name == "idle" then
+    -- If we interrupted a patient at the front of the queue they get use_object instead of idle
+    if queue_front:getCurrentAction().name == "idle" or queue_front:getCurrentAction().name == "use_object" then
```

Comment:
I been told not to try and interfere with the action queue and handle the exception instead.

Reply:
Though saying that if someone wants to work out why I have no complaints